app/views/home/_active_link_change.html.haml

```diff
@@ -0,0 +1,14 @@
+:javascript
```

Comment:
I had to append this script at the end of the file to ensure that it is being loaded even after page re-navigation.